catroid/src/main/java/org/catrobat/catroid/ProjectManager.java

```diff
@@ -483,4 +485,34 @@ public void setCurrentlyEditedScene(Scene scene) {
 		currentlyEditedScene = scene;
 		currentlyPlayingScene = scene;
 	}
+
+	public void createHeader(Project mergedProject, Project firstProject, Project secondProject) {
```

Comment:
please rename this to "mergeHeaders" or something similar. ProjectManager is such a big class and "createHeader" isnt going to be clear to anyone that this has to do with merge.

Reply:
Done.